README.md

```diff
@@ -147,13 +147,18 @@ deployed Performance Operator and configured MCP and nodes. It will create its o
 The latency-test container image gives the possibility to run the latency 
 test without need to install go, ginkgo or other go related modules.
 
-The test himself is running the `oslat` binary and verifies if the maximal latency returned by the `oslat`
-less than specified value under the `OSLAT_MAXIMUM_LATENCY`.
+The test itself is running the `oslat` `cyclictest` and `hwlatdetect` binaries and verifies if the maximal latency returned by each one of the tools is
+less than specified value under the `MAXIMUM_LATENCY`.
 
-To run the latency test inside of the container:
+To run the latency test inside the container:
 
 ```
-docker run --rm -v /kubeconfig:/kubeconfig -e KUBECONFIG=/kubeconfig -e LATENCY_TEST_RUN=true -e LATENCY_TEST_RUNTIME=60 -e OSLAT_MAXIMUM_LATENCY=700 alukiano/latency-test:4.6-snapshot /usr/bin/run-tests.sh
+docker run --rm -v /kubeconfig:/kubeconfig \
+-e KUBECONFIG=/kubeconfig \
+-e LATENCY_TEST_RUN=true \
+-e LATENCY_TEST_RUNTIME=60 \
+-e MAXIMUM_LATENCY=700 \
+ quay.io/openshift-kni/cnf-tests /usr/bin/run-tests.sh
```

Comment:
I'm not sure this is image that suppose to be shown here

Reply:
It is the one available to the public, so I do not see a problem with it.